package.json

```
@@ -22,7 +22,7 @@
     "@aws-crypto/client-node": "^3.1.1",
     "@contentful/rich-text-react-renderer": "^15.11.1",
     "@elastic/elasticsearch": "7.10",
-    "axios": "^0.27.2",
+    "axios": "^1.6.0",
```

may as well pin this version

```suggestion
    "axios": "1.6.0",
```